saleor/graphql/product/bulk_mutations/product_variant_bulk_create.py

```diff
@@ -864,7 +865,13 @@ def save_variants(cls, variants_data_with_errors_list, product):
 
             if not variant:
                 continue
-
+            track_inventory_by_default = get_track_inventory_by_default(info)
+            if track_inventory_by_default is not None:
+                variant.track_inventory = (
+                    track_inventory_by_default
+                    if variant_data["cleaned_input"].get("track_inventory") is None
+                    else variant_data["cleaned_input"].get("track_inventory")
+                )
```

Comment:
It could be simplified
```suggestion
                track_inventory = variant_data["cleaned_input"].get("track_inventory")
                variant.track_inventory = track_inventory or track_inventory_by_default
```